optuna/integration/wandb.py

```diff
@@ -84,11 +89,32 @@ def __init__(
 
     def __call__(self, study: optuna.study.Study, trial: optuna.trial.FrozenTrial) -> None:
 
-        direction = study.direction.name
-        attributes = {"direction": direction}
+        if isinstance(self._metric_name, list):
+            if len(self._metric_name) != len(trial.values):
+                raise RuntimeError(
+                    "Running multi-objective optimization "
+                    "with {} objective values, but {} names specified. "
+                    "Match objective values and names, or use default broadcasting.".format(
+                        len(trial.values), len(self._metric_name)
+                    )
+                )
+
+            else:
+                names = self._metric_name
+
+        if isinstance(self._metric_name, str):
+            if len(trial.values) > 1:
+                # broadcast default name for multi-objective optimization
+                names = ["{}_{}".format(self._metric_name, i) for i in range(len(trial.values))]
```

Comment:
I think we should write this "broadcast" logic in the document.

Reply:
In addition, how about testing this logic?

Reply:
Hi @HideakiImamura, thanks for the feedback! We are testing this logic in 72f7071b1a51d6ae082071bf2165a0e9a60a1b51. Do you think there should be more test cases covering this?

Reply:
Pushed documentation update on broadcasting in 5cb2d80f229bd305437bfda39984e3038a859f3c.

Reply:
It certainly looks like it has been tested enough. Thanks!